ingest/lambdas/bulk-ingester.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+const bulkIngester = require('../lib/bulk-ingester');
+
+const handler = async () => {
+  const dspaceEndpoint = process.env.DSPACE_ENDPOINT;
+  const ingestTopicArn = process.env.INGEST_TOPIC_ARN;
+  if (dspaceEndpoint === undefined || ingestTopicArn === undefined) {
+    console.log('WARN: Bulk ingester called without required environment variables.');
+    return;
+  }
+
+  bulkIngester(
+    dspaceEndpoint,
+    ingestTopicArn
+  );
+};
+
+module.exports = handler;
```

Comment:
Lambda is going to invoke this as `bulkIngester.handler`, so we need to make sure to export an object containing a handler function rather than exporting a function directly.

```suggestion
module.exports = { handler };
```